sql/upgrade/sl3.0.sql

```diff
@@ -461,12 +474,34 @@ UPDATE :slschema.vendor SET entity_id = (SELECT id FROM entity WHERE 'V-' || ven
 
 UPDATE :slschema.customer SET entity_id = coalesce((SELECT min(id) FROM entity WHERE 'C-' || customernumber = control_code), entity_id);
 
+--TODO: Do we need that for MC?
+INSERT INTO defaults(setting_key,value)
+    SELECT 'curr',array_to_string(array_agg(curr),':')
+    FROM :slschema.curr
+    WHERE rn=1;
```

Comment:
I don't think this statement is correct, because it generates a string of the format "CAD:USD:EUR". Under "mc", the value of the "curr" key is the default currency code. Not all of them.

Reply:
The code was producing the proper currency because of the rn restriction. Nevertheless, there is no need to aggregate a singleton array.